store/interfaces.json

```diff
@@ -0,0 +1,7 @@
+{
+  "footer": {
+    "allowed": [],
+    "required": [],
+    "inheritComponent": true
```

Comment:
It doesn't exist in *interfaces.json*.

If you set `component` here, you will need to set `inheritComponent: true` in *blocks.json*.